docs/github/contribute_to_repos.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to data services repos
+
+These guidelines are meant to provide a foundation for collaboration in Cal-ITP's data services repos (primarily [#data-infra](https://github.com/cal-itp/data-infra).)
+
+## Issues:
+* When submitting an issue, please make it clear what the acceptance criteria are; i.e., what is sufficient to consider a fix “done” for the issue.
+    * For example: The following issue is vague and could fall victim to an unhelpful cycle of swirl and scope creep:
+      > Add best practices to reports
+
+    * The following issue is clearer. If scope were to increase (for example, the author realizes that they also want a chart), it would be clearer that that should be created as a new, follow-up request.
+       > Add a new section to reports called ‘Best Practices Violations’. It should be inserted immediately after and appear just like the existing critical validation errors table and include validation notices W1, W2, and W3.
```

Comment:
I support this as a good framework for creating focused GitHub issues, however, I think it'd be a good idea to also note the proper venue for proposing larger changes that need to be scoped out.

Reply:
Added in https://github.com/cal-itp/data-infra/pull/1288/commits/920ee0ba31aa713971e1e8ee8b85ca85c8206819 and clarified purpose of this expectation in https://github.com/cal-itp/data-infra/pull/1288/commits/213c4b5ec4c5bf61370bf405b2af9ee3cd482cb4. Please let me know if additional context or clarification is needed.

Reply:
Works for me